common/spring-cloud-stream-modules-common-configuration/src/test/java/org/springframework/cloud/stream/module/ftp/FtpSessionFactoryPropertiesTests.java

```diff
@@ -98,6 +99,7 @@ public void noPropertiesThrowsMeaningfulException() {
 			context.register(Conf.class);
 			context.refresh();
 			fail("should throw exception");
+			context.close();
```

Comment:
Is this reachable? Seems like in either case an exception is thrown before getting here. Maybe we should close in the `finally` clause.